JiraPS.build.ps1

```diff
@@ -194,6 +194,13 @@ task UpdateManifest GetNextVersion, {
     if ($ModuleAlias) {
         BuildHelpers\Update-Metadata -Path "$env:BHBuildOutput/$env:BHProjectName/$env:BHProjectName.psd1" -PropertyName AliasesToExport -Value @($ModuleAlias.Name)
     }
+
+    $Prerelease = ''
+    if ("$env:BHBranchName" -notin @('master','main')) {
+        $Prerelease = "$env:BHBranchName".ToLower() -replace '[^a-zA-Z0-9]'
```

Comment:
I would like more documentation on this.
I would expect `$Prerelease` to be empty or `-eq "alpha"`. But I don't know how you would name branches to get to that value

https://learn.microsoft.com/en-us/powershell/gallery/concepts/module-prerelease-support?view=powershellget-3.x#identifying-a-module-version-as-a-prerelease

> * Prerelease string may only be specified when the ModuleVersion is 3 segments for Major.Minor.Build. This aligns with SemVer v1.0.0.
> * A hyphen is the delimiter between the Build number and the Prerelease string. A hyphen may be included in the Prerelease string as the first character, only.
> * The Prerelease string may contain only ASCII alphanumerics [0-9A-Za-z-]. It is a best practice to begin the Prerelease string with an alpha character, as it will be easier to identify that this is a prerelease version when scanning a list of packages.
> * Only SemVer v1.0.0 prerelease strings are supported at this time. Prerelease string must not contain either period or + [.+], which are allowed in SemVer 2.0.
> * Examples of supported Prerelease string are: -alpha, -alpha1, -BETA, -update20171020